### ISSUE TITLE
Any plan to support for rain and snow?

### ISSUE THREAD

**Issue by mrzashouri** [esmini/esmini#602]

**Comment by mrzashouri** [comment]
Yes, this is the use case. I am working in an OEM. We are evaluating usage of Esmini for virtual testing of ADAS with ideal sensors in the loop.

**Comment by eknabevcc** [comment]
Ok, I see. Thanks for clarification. Our plan with Environment actions is focusing on feeding ideal sensor data, while visualization might be very simplified,  (just to indicate what's going on in the scenario - probably far from photo-realism).

Unfortunately we don't have a release time for this feature, but it's good to hear about need since it affects prioritizations.
